tests/terraform/graph/graph_builder/test_local_graph.py

```diff
@@ -108,6 +109,39 @@ def test_set_variables_values_from_modules(self):
             else:
                 self.assertEqual(var_value, default_val)
 
+    def test_definition_creation_by_dirs(self):
```

Comment:
nice test!
However, I still think we need some more for the logic - maybe add 1 with more than 1 created module.
you can find a good example under `tests/terraform/graph/resources/modules`.
This one even utilizes different modules with a reference to other directories, so I feel it will be a very strong test.
(You don't have to take the whole folder as it might be also too big, maybe just one sub directory like `stacks`)